backend/qemu.pm

```diff
@@ -545,16 +545,20 @@ sub start_qemu {
                 # XXX: workaround for OVMF wanting to write NVvars into first FAT partition
                 # we need to replace -bios with proper pflash drive specification
                 $params[-1] .= ',snapshot=on' if $vars->{UEFI};
-                push(@params, '-device', "$vars->{CDMODEL},drive=cd0,bus=scsi0.0");
+                my $bus //= '';
```

Comment:
true,  tried to avoid but just = is enough :) I tried also similar fancy way but it didn't work at the and
```
Use of uninitialized value $bus in concatenation (.) or string at /usr/lib/os-autoinst/backend/qemu.pm line 549.
Use of uninitialized value $bus in concatenation (.) or string at /usr/lib/os-autoinst/backend/qemu.pm line 559.
```